templates/Apple/Applesauce WOZ.tcl

```diff
@@ -0,0 +1,177 @@
+# Apple/Applesauce WOZ.tcl
+# Binary format for Apple II disk images in the Applesauce WOZ format.
+# https://applesaucefdc.com/woz/reference2/
+# 2022 Aug 13 | chris-torrence | Initial implementation
+
+requires 0 "574F5A" ;# WOZ
+requires 4 "FF0A0D0A" ;# check bytes
+set woz [ascii 4]
+ascii 4
+entry "Signature" $woz 4 0
+entry "Checkbytes" "FF0A0D0A" 4 4
+# TODO: Should probably verify the crc32
+set crc32 [uint32 -hex "CRC32"]
+
+# Return current file position - offset
+# Useful when tying data chunks to their file location.
+proc p {offset} {
```

Comment:
a more description name would be better here.

Reply:
good point - I was going for brevity but better to give it an obvious name - I'll change it